crates/ruff_linter/src/rules/pep8_naming/rules/invalid_first_argument_name.rs

```diff
@@ -127,19 +127,28 @@ impl Violation for InvalidFirstArgumentNameForMethod {
 #[derive(ViolationMetadata)]
 pub(crate) struct InvalidFirstArgumentNameForClassMethod {
     argument_name: String,
+    // Whether the method is `__new__`
+    is_new: bool,
 }
 
 impl Violation for InvalidFirstArgumentNameForClassMethod {
     const FIX_AVAILABILITY: ruff_diagnostics::FixAvailability =
         ruff_diagnostics::FixAvailability::Sometimes;
 
     #[derive_message_formats]
+    // The first string below is what shows up in the documentation
+    // in the rule table, and it is the more common case.
+    #[allow(clippy::if_not_else)]
```

Comment:
Lol, yet another reason for us to introduce a `summary_sentence()` method to the Violation trait instead of deriving the summary sentence from the `message()` method 🙃